tracker/view/add.py

```diff
@@ -199,4 +200,21 @@ def add_group():
 
     db.session.commit()
     flash('Added {}'.format(group.name))
+    missing_lib32_variant(pkgnames, group)
```

Comment:
This doesn't quite say what it does, either we have a ```get_missing...``` and loop+flash here or we name it like ```flash_missing...``` or such. ```missing_lib32_variant``` itself doesn't say much what it actually does.